frame/support/src/dispatch.rs

```diff
@@ -60,6 +61,28 @@ pub trait Callable<T> {
 // https://github.com/rust-lang/rust/issues/51331
 pub type CallableCallFor<A, R> = <A as Callable<R>>::Call;
 
+/// Origin for the System pallet.
```

Comment:
```suggestion
/// Inherent origin of FRAME
```

Or probably something better.